response_operations_ui/templates/create-message.html

```diff
@@ -0,0 +1,112 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+<style>
+    table, th, td {
+    border: 0px;
+    }
+</style>
+
+{% if form.errors %}
+    <div class="panel panel--error">
+        <div class="panel__header">
+            {% if form.errors|length > 1 %}
+                <h1 class="panel__title venus">This page has {{ form.errors|length }} errors</h1>
+            {% elif form.errors %}
+                <h1 class="panel__title venus">This page has {{ form.errors|length }} error</h1>
+            {% endif %}
+        </div>
+        <div class="panel__body">
+            <ol>
+                {% for error in form.errors %}
+                <li class="panel__title venus">{{ form.errors[error][0] }}</li>
```

Comment:
Can a field produce multiple errors at the same time?  If so then this'll need to change.  If not, then theres no issue.